core/src/OpenACC/Kokkos_OpenACC.hpp

```diff
@@ -65,6 +65,10 @@ static_assert(false,
 #include <iosfwd>
 #include <string>
 
+// Below macro is temporarily enabled to avoid issues on existing OpenACC
+// compilers not supporting lambda with parallel loops.
+#define KOKKOS_ENABLE_OPENACC_COLLAPSE_HIERARCHICAL_CONSTRUCTS
```

Comment:
Should this be a FIXME?

Reply:
I'll leave this decision up to you.

Reply:
`FIXME_OPENACC` is added.
(This implementation is valid but non-portable solution, which works only for NVHPC.)